.github/workflows/build.yml

```diff
@@ -17,22 +17,22 @@ jobs:
     steps:
       - name: Checkout PR
         if: github.event_name == 'pull_request'
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4.1.1
```

Comment:
same as in https://github.com/CoreMedia/content-hub-adapter-rss/pull/12#pullrequestreview-1811070817